solitude/base.py

```diff
@@ -719,4 +738,14 @@ class NonDeleteModelViewSet(
     Same as the ModelViewSet, without the DeleteMixin. Uses our local mixins
     to give us ETag support.
     """
-    pass
+
+    # Allow tastypie resources to connect to viewsets, to delete
+    # once tastypie is gone. You really don't want to know.
+    def get_via_uri(self, uri, request=None):
```

Comment:
And this one, too: https://github.com/mozilla/zamboni/commit/ba7e3a5217fc3054154ba5f37cf1b01e17a87f91